README.md

```diff
@@ -0,0 +1,55 @@
+# Week-7-Assignment
+Week 7 Assignment for Intro to Programming
+# This is a header for the application
+# You should read this header and insert your name and your date below as part of the peer review
+# This is a typical part of any program
+# Author: Darriane "Ari" Elledge
+# Creation Date: 12/2/2020
+# Below is a simple program with 10 issues (some are syntax errors and some are logic errors.  You need to identify the issues and correct them.
+
+import random
+import time
+
+def displayIntro():
+	print('''You are in a land full of dragons. In front of you,
+	you see two caves. In one cave, the dragon is friendly
+	and will share his treasure with you. The other dragon
+	is greedy and hungry, and will eat you on sight.''')
+	print()
+
+def chooseCave():
+    cave = ''
+	while cave != '1' and cave != '2':
+		print('Which cave will you go into? (1 or 2)')
+		cave = input()
+
+	return caves
+
+def checkCave(chosenCave):
+	print('You approach the cave...')
+	#sleep for 2 seconds
+	time.sleep(2)
+	print('It is dark and spooky...')
+	#sleep for 2 seconds
+	time.sleep(3)
```

Comment:
if this is supposed to be for 2 seconds, as denoted in the comments, this should be a 2 again, not 3.